Makefile

```diff
@@ -42,8 +42,16 @@ publish-local:
 	export IS_LOCAL_DEVELOPMENT=true; ./gradlew :libandroid-navigation-ui:uploadArchives
 
 graphs:
-	./gradlew :libandroid-navigation:generateDependencyGraphMapboxLibraries
-	./gradlew :libandroid-navigation-ui:generateDependencyGraphMapboxLibraries
+	./gradlew :libnavigator:generateDependencyGraphMapboxLibraries
+	./gradlew :libnavigation-base:generateDependencyGraphMapboxLibraries
+	./gradlew :libdirections-onboard:generateDependencyGraphMapboxLibraries
+	./gradlew :libdirections-offboard:generateDependencyGraphMapboxLibraries
+	./gradlew :libdirections-hybrid:generateDependencyGraphMapboxLibraries
+	./gradlew :libnavigation-metrics:generateDependencyGraphMapboxLibraries
+	./gradlew :libtrip-notification:generateDependencyGraphMapboxLibraries
+	./gradlew :libnavigation-util:generateDependencyGraphMapboxLibraries
+	./gradlew :libnavigation-core:generateDependencyGraphMapboxLibraries
+	./gradlew :libnavigation-ui:generateDependencyGraphMapboxLibraries
```

Comment:
NIT Could we add separate commands for pre-1.0 and 1.0? Similar to other commands 👇